packages/demos/README.md

```diff
@@ -14,10 +14,18 @@ yarn
 
 Now run one of the following demos:
 
+```sh
+yarn watch:demo async-ssr-manager
+```
+
 ```sh
 yarn watch:demo history-service
 ```
 
+```sh
+yarn watch:demo history-service-server-request
```

Comment:
The name of the demo does not give me an explanation why this demo exists. Couldn't you extend the existing one? If there are good reasons (I suppose), maybe a short reason/description in the README would be appropriate.

Reply:
I couldn't use the existing one because we need to make sure, no client-side code (e.g. the browser history) changes the server-side rendered HTML that we want to observe.

Reply:
Hm, maybe [`page.setJavaScriptEnabled`](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagesetjavascriptenabledenabled) could help with that?

Reply:
yes, maybe that could be an option

Reply:
done